examples/denoise_covar2d.py

```diff
@@ -21,14 +20,14 @@
 from aspire.image import Image
 from aspire.utils.matrix import anorm
 from aspire.utils.matlab_compat import randn
-from aspire.denoise.covar2d import RotCov2D
-from aspire.denoise.covar2d_ctf import Cov2DCTF
+from aspire.estimation.covar2d import RotCov2D
 from aspire.utils.blk_diag_func import blk_diag_add, blk_diag_mult, blk_diag_norm
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger('aspire')
```

Comment:
Why this change?

Reply:
We want to use a more consistent format for logging as the scripts in  `src/aspire/commands/`.

Reply:
Ok.